### PR TITLE
Focus readme files only when preview pane is open

### PR DESCRIPTION
**Pull request by d2dyno1**
**Resolved / Related Issues**
\-

**Details of Changes**
A readme file will be selected only when preview pane is opened.

**Validation**
How did you test these changes?
- [x] Built and ran the app

### CODE REVIEW

**Comment by winston-de** [comment]
May not be caused by this PR, but if another item contains the term readme, it will be selected instead of readme.md on navigated. Eg, a folder named readme_assets will be selected instead of readme.md.

**Review by winston-de — APPROVED** [review]

File: Files/Views/MainPage.xaml.cs [hunk 1]
```diff
@@ -28,6 +28,8 @@ namespace Files.Views
     /// </summary>
     public sealed partial class MainPage : Page, INotifyPropertyChanged
     {
+        public static MainPage Instance => (MainPage)((Frame)Window.Current.Content).Content;
```
Comment (yaichenbaum) [inline Files/Views/MainPage.xaml.cs:31]: Is this change related to the auto preview of readme files?

Comment (d2dyno1) [reply Files/Views/MainPage.xaml.cs:31]: Can be removed

**Review by yaichenbaum — APPROVED** [review]
